templates/blocks/content_four_column.twig

```diff
@@ -57,7 +57,7 @@
 				{% if ( posts|length <= 4
 					or ( posts|length > 4 and posts|length <= 8 and posts_view == '3' )
```

Comment:
Looks good. can you change these checks also (for 2 rows)? they seems wrong
`posts|length > 4 and posts|length <= 8 and posts_view == '3' `

It should be posts_view=='2', since the values have different meaning between the old blocks and the new.